api/src/main/java/org/pmiops/workbench/db/dao/UserService.java

```diff
@@ -210,6 +211,18 @@ public User apply(User user) {
     });
   }
 
+  public User setEraCommonsStatus(String linkedNihUsername, Timestamp linkExpireTime, Timestamp completionTime) {
```

Comment:
I'm a little unclear on the purpose of this method. In the syncEraCommonsStatus method on ProfileController, we simply call userDao.save(user). Is there a reason to have this separate method set up? (If so... it's worth a method-level comment to help guide the user.)

Just looking around at other methods in this file... one thing that ended up happening for the compliance training module was that we wanted to have ProfileController and UserController call a single service-level method to execute the sync for a given user. So the full logic for syncing a module's state lives within UserService. That's not an unreasonable way to approach this – but would involve moving all of the logic from ProfileController to here.

Reply:
Agreed. Doing that.